editoast/src/railjson/operation/update.rs

```diff
@@ -186,4 +205,115 @@ mod test {
+                infra.id
+            ))
+            .load::<SightDistance>(&conn).unwrap();
+            assert_eq!(updated_length[0].sight_distance, 15.0);
+
+            Ok(())
+        });
+    }
+
+    #[test]
+    fn valide_update_signal_optionnal() {
+        let conn = PgConnection::establish(&PostgresConfig::default().url()).unwrap();
+        conn.test_transaction::<_, Error, _>(|| {
+            let signal_creation = RailjsonObject::Signal {
+                railjson: Signal {
+                    ..Default::default()
+                },
+            };
+            let infra = Infra::create(&"test".to_string(), &conn).unwrap();
+            assert!(apply_create_operation(&signal_creation, infra.id, &conn).is_ok());
+
+            let update_signal = UpdateOperation {
+                obj_id: "my_signal".to_string(),
+                obj_type: ObjectType::Signal,
+                railjson_patch: from_str(
+                    r#"[
+                    { "op": "replace", "path": "/comment", "value": "Test Passed" }
+                  ]"#,
+                )
+                .unwrap(),
+            };
+
+            assert!(update_signal.apply(infra.id, &conn).is_ok());
+
+            let updated_comment = sql_query(format!(
+                "SELECT (data->>'comment') as comment FROM osrd_infra_signalmodel WHERE obj_id = 'my_signal' AND infra_id = {}",
+                infra.id
+            ))
+            .load::<Comment>(&conn).unwrap();
+            assert_eq!(updated_comment[0].comment, "Test Passed");
+
+            Ok(())
+        });
+    }
+
+    #[test]
+    fn valide_update_speed() {
+        let conn = PgConnection::establish(&PostgresConfig::default().url()).unwrap();
+        conn.test_transaction::<_, Error, _>(|| {
+            let speed_creation = RailjsonObject::SpeedSection {
+                railjson: SpeedSection {
+                    id: "my_speed".to_string(),
+                    speed: 100.0,
+                    ..Default::default()
+                },
+            };
+
+            let infra = Infra::create(&"test".to_string(), &conn).unwrap();
+            assert!(apply_create_operation(&speed_creation, infra.id, &conn).is_ok());
+
+            let update_speed = UpdateOperation {
+                obj_id: "my_speed".to_string(),
+                obj_type: ObjectType::SpeedSection,
+                railjson_patch: from_str(
+                    r#"[
+                    { "op": "replace", "path": "/speed", "value": 80.0 }
+                  ]"#,
+                )
+                .unwrap(),
+            };
+
+            assert!(update_speed.apply(infra.id, &conn).is_ok());
+
+            let updated_speed = sql_query(format!(
+                "SELECT (data->>'speed')::float as speed FROM osrd_infra_speedsectionmodel WHERE obj_id = 'my_speed' AND infra_id = {}",
+                infra.id
+            ))
+            .load::<Speed>(&conn).unwrap();
```

Comment:
Can't you use `get_result` method instead of `load` since we expect having only one row returned?